ocs_ci/helpers/managed_services.py

```diff
@@ -3,6 +3,7 @@
 """
 import logging
 import yaml
+from semantic_version import Version
```

Comment:
use version module. same as in other places. https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/utility/version.py

Reply:
done